app/views/admin/participants/induction_records/edit_training_status.html.erb

```diff
@@ -0,0 +1,17 @@
+<% content_for :before_content, govuk_back_link(text: "Back", href: admin_participant_path(@participant_profile)) %>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-l">Change the training status for this participant at <%= @induction_record.school_name %></h1>
+    <%= form_for @induction_record, url: update_training_status_admin_participant_induction_records_path(@participant_profile, @induction_record), method: :put do |f| %>
+      <%= f.govuk_collection_select :training_status,
+                                    training_statuses,
+                                    :id,
+                                    :value,
+                                    label: { text: "Select training status" },
+                                    options: { selected: @induction_record.training_status }
+      %>
```

Comment:
Can we make this radio buttons? (Easier to use and less error-prine - especially when there's only 2 options!)

Reply:
Sure, I will convert it